### PR TITLE
Potential fix for issue 307

### PR DESCRIPTION
**Pull request by jthrun**
https://github.com/smartdevicelink/sdl_android/issues/307.

One possible way of propagating the broadcasting router info down to the transport connection.

### CODE REVIEW

**Comment by joeygrover** [comment]
I'll take a look at this tomorrow, but I think you're on the right path from just looking at the changes.

File: sdl_android_lib/src/com/smartdevicelink/transport/SdlBroadcastReceiver.java [hunk 1]
```diff
@@ -31,6 +31,8 @@
 		
 	public static ComponentName runningBluetoothServicePackage = null;
 
+	private static ComponentName broadcastServicePackage = null;
```
Comment (joeygrover) [inline sdl_android_lib/src/com/smartdevicelink/transport/SdlBroadcastReceiver.java]: You don't need to store this, have the TransportBroker already has this info

**Comment by joeygrover** [comment]
You should have the TransportBroker just supply the component name of the service it's bound to [here](https://github.com/smartdevicelink/sdl_android/blob/develop/sdl_android_lib/src/com/smartdevicelink/transport/TransportBroker.java#L163) using this [variable](https://github.com/smartdevicelink/sdl_android/blob/develop/sdl_android_lib/src/com/smartdevicelink/transport/TransportBroker.java#L47) .

**Comment by jthrun** [comment]
Joey,
I've updated the code per your suggestions.  Can you take another look?

**Comment by joeygrover** [comment]
Keeping this open until #312 is reviewed and accepted, but it should take care of the issue this PR solves.

**Comment by joeygrover** [comment]
#312 was merged and #307 was confirmed to be solved. Closing this PR.

**Pull request closed without merging**